src/test/java/org/elasticsearch/indices/template/SimpleIndexTemplateTests.java

```diff
@@ -54,6 +56,7 @@
 /**
  *
  */
+@ClusterScope(scope = ElasticsearchIntegrationTest.Scope.TEST)
```

Comment:
Right, I should have thought of that... I'll add a IndexTemplateBlocksTests class.